runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/decompResolveFrame/decomp003.c

```diff
@@ -22,6 +22,7 @@
 
 #include "jvmti_test.h"
 #include <string.h>
+#include <strings.h>
```

Comment:
@Deigue  Given that this is causing failure - I wonder, we should guard this with `J9ZOS` same as how `atoe.h` was done (Assuming that only z/OS needs the function defined in `strings.h`which you may have bumped into with OpenXL work.

Reply:
yes that sounds like a good idea, I think the build must have failed on one of the platforms we didnt verify with.